src/pages/ReportParticipantsPage.tsx

```diff
@@ -1,135 +1,355 @@
+
+    /** Opens the member details page */
+    const openMemberDetails = useCallback(
+        (item: MemberOption) => {
+            if (isGroupChat && isCurrentUserAdmin) {
+                Navigation.navigate(ROUTES.REPORT_PARTICIPANTS_DETAILS.getRoute(report.reportID, item.accountID));
+                return;
+            }
+            Navigation.navigate(ROUTES.PROFILE.getRoute(item.accountID));
+        },
+        [report, isCurrentUserAdmin, isGroupChat],
+    );
+    const headerTitle = useMemo(() => {
+        if (
+            ReportUtils.isChatRoom(report) ||
+            ReportUtils.isPolicyExpenseChat(report) ||
+            ReportUtils.isChatThread(report) ||
+            ReportUtils.isTaskReport(report) ||
+            ReportUtils.isMoneyRequestReport(report)
+        ) {
+            return translate('common.members');
+        }
+        if (isGroupChat) {
+            return translate('common.everyone');
+        }
+
+        return translate('common.details');
+    }, [report, translate, isGroupChat]);
     return (
         <ScreenWrapper
             includeSafeAreaPaddingBottom={false}
+            style={[styles.defaultModalContainer]}
             testID={ReportParticipantsPage.displayName}
+            shouldShowOfflineIndicatorInWideScreen
         >
-            {({safeAreaPaddingBottomStyle}) => (
-                <FullPageNotFoundView shouldShow={!report || ReportUtils.isArchivedRoom(report) || ReportUtils.isSelfDM(report)}>
-                    <HeaderWithBackButton
-                        onBackButtonPress={report ? () => Navigation.goBack(ROUTES.REPORT_WITH_ID_DETAILS.getRoute(report.reportID)) : undefined}
-                        title={translate(
-                            ReportUtils.isChatRoom(report) ||
-                                ReportUtils.isPolicyExpenseChat(report) ||
-                                ReportUtils.isChatThread(report) ||
-                                ReportUtils.isTaskReport(report) ||
-                                ReportUtils.isMoneyRequestReport(report)
-                                ? 'common.members'
-                                : 'common.details',
-                        )}
+            <FullPageNotFoundView shouldShow={!report || ReportUtils.isArchivedRoom(report) || ReportUtils.isSelfDM(report)}>
+                <HeaderWithBackButton
+                    title={headerTitle}
+                    subtitle={isGroupChat ? translate('common.members') : ''}
+                    icon={isGroupChat ? Illustrations.ReceiptWrangler : undefined}
+                    onBackButtonPress={report ? () => Navigation.goBack(ROUTES.REPORT_WITH_ID_DETAILS.getRoute(report.reportID)) : undefined}
+                    shouldShowBackButton
+                    guidesCallTaskID={CONST.GUIDES_CALL_TASK_IDS.WORKSPACE_MEMBERS}
+                    subtitleOnTop={isGroupChat}
+                >
+                    {!isSmallScreenWidth && headerButtons}
```

Comment:
I'm a bit confused why `headerButtons` lives in `HeaderWithBackButton` on wide screens but in its own view on narrow screens.